service/api_logic/news_logic.py

```diff
@@ -20,24 +19,29 @@ def fetch_news(session, order_by: ClauseElement = None, limit: int = None, filte
 
 
 def get_news_by_count(count: int, session):
-    news = fetch_news(session, order_by=News.save_at.desc(), limit=count)
-    return json_news(news)
+    try:
+        news = fetch_news(session, order_by=News.save_at.desc(), limit=count)
+        return json_news(news)
+    except Exception:
```

Comment:
except-raise again

exception won't be handled this way.

By the way: I wonder what would happen if I pass negative number here :)

Do validation of the incoming data. That is usually done in the API layer